c2corg_ui/static/js/documentediting.js

```diff
@@ -231,8 +230,17 @@ app.DocumentEditingController.prototype.successRead_ = function(response) {
   this.scope_[this.modelName_] = data;
 
   if (this.modelName_ === 'outing') {
-    this.scope_['outing'] = this.formatOuting_(this.scope_['outing']);
-    this.differentDates = app.utils.areDifferentDates(this.scope_['outing']['date_start'], this.scope_['outing']['date_end']);
+    var outing = this.scope_['outing'];
+    // check if user has right to edit -> the user is one of the associated users
+    if (this.auth_.hasEditRightsOuting(outing['associations']['users'])) {
+      this.scope_['outing'] = this.formatOuting_(outing);
+      this.differentDates = app.utils.areDifferentDates(outing['date_start'], outing['date_end']);
+    } else {
+      this.alerts_.addError('you have no rights to edit this document');
+      setTimeout(function () {
+        window.location.href = '/';
```

Comment:
It would be better to redirect to the document view page rather than to the homepage.